consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/QbftBlockHeaderValidationRulesetFactoryTest.java

```diff
@@ -60,7 +61,8 @@ public void bftValidateHeaderPasses() {
             .buildHeader();
```

Comment:
Can you add a test that shows that the header validation passes when we have header that contains a base fee